pkgs/top-level/all-packages.nix

```
@@ -4251,6 +4251,10 @@ in
 
   xkcdpass = with python3Packages; toPythonApplication xkcdpass;
 
+  xjobs = callPackage ../tools/misc/xjobs {
+    inherit installShellFiles flex ncurses which;
+  };
```

```suggestion
  xjobs = callPackage ../tools/misc/xjobs { };
```

This is done automatically.